components/Drop/DropProgress.tsx

```diff
@@ -0,0 +1,77 @@
+import { Flex, Progress, SimpleGrid, Text } from '@chakra-ui/react'
+import BigNumber from 'bignumber.js'
```

Comment:
We usually import BigNumber from ethers:
```suggestion
import { BigNumber } from '@ethersproject/bignumber'
```
If you really need the `bignumber.js` version to handle decimals, then we usually import it like:
```
import { BigNumber as BN } from 'bignumber.js'
```

Reply:
[Remove bignumber.js imports](https://github.com/liteflow-labs/starter-kit/pull/359/commits/80d377f017d0d7ebae6617c8c386a018de3c7e6a)